Type definitions are wrong for REST helper

#### What are you trying to achieve?

Use the REST helper in a typescript project

#### What do you get instead?

Typescript compile errors because the provided type definitions for the REST helper do not match the actual method signatures.

The _executeRequest method and all of the send* methods are async and return the response object, however the type definition marks them as returning void. This means that in my code using these methods if I actually try to use the response object, typescript throws an error: `Type 'void' is not assignable to type 'AxiosResponse<any>'`

In addition, the arguments for each method are all marked as being required, even though most of them are optional, which means I have to pass `undefined` for each argument to prevent typescript compiler errors. 

> Example showing both of the above issues
```ts
import { AxiosResponse } from 'axios';
const res: AxiosResponse = await this.sendPostRequest('', {});
```

```bash
helpers/steps_file.ts:46:13 - error TS2322: Type 'void' is not assignable to type 'AxiosResponse<any>'.

46       const res: AxiosResponse = await this.sendPostRequest('', {});
               ~~~

helpers/steps_file.ts:46:45 - error TS2554: Expected 3 arguments, but got 2.

46       const res: AxiosResponse = await this.sendPostRequest('', {});
                                               ~~~~~~~~~~~~~~~~~~~~~~~
```

### Details

* CodeceptJS version: 2.6.10
* NodeJS Version: 14.11.0
* Operating System: Windows 10

--- a/docs/helpers/REST.md
+++ b/docs/helpers/REST.md
@@ -61,6 +61,8 @@ Executes axios request
 
 -   `request` **any** 
 
+Returns **[Promise][2]&lt;any>** response
+
 ### _url
 
 Generates url based on format sent (takes endpoint + url if latter lacks 'http')
@@ -80,7 +82,9 @@ I.sendDeleteRequest('/api/users/1');
 #### Parameters
 
 -   `url` **any** 
--   `headers` **[object][2]** the headers object to be sent. By default it is sent as an empty object 
+-   `headers` **[object][3]** the headers object to be sent. By default it is sent as an empty object 
+
+Returns **[Promise][2]&lt;any>** response
 
 ### sendGetRequest
 
@@ -93,7 +97,9 @@ I.sendGetRequest('/api/users.json');
 #### Parameters
 
 -   `url` **any** 
--   `headers` **[object][2]** the headers object to be sent. By default it is sent as an empty object 
+-   `headers` **[object][3]** the headers object to be sent. By default it is sent as an empty object 
+
+Returns **[Promise][2]&lt;any>** response
 
 ### sendPatchRequest
 
@@ -108,9 +114,11 @@ I.sendPatchRequest('/api/users.json', secret({ "email": "user@user.com" }));
 
 #### Parameters
 
--   `url` **[string][3]** 
+-   `url` **[string][4]** 
 -   `payload` **any** the payload to be sent. By default it is sent as an empty object 
--   `headers` **[object][2]** the headers object to be sent. By default it is sent as an empty object 
+-   `headers` **[object][3]** the headers object to be sent. By default it is sent as an empty object 
+
+Returns **[Promise][2]&lt;any>** response
 
 ### sendPostRequest
 
@@ -127,7 +135,9 @@ I.sendPostRequest('/api/users.json', secret({ "email": "user@user.com" }));
 
 -   `url` **any** 
 -   `payload` **any** the payload to be sent. By default it is sent as an empty object 
--   `headers` **[object][2]** the headers object to be sent. By default it is sent as an empty object 
+-   `headers` **[object][3]** the headers object to be sent. By default it is sent as an empty object 
+
+Returns **[Promise][2]&lt;any>** response
 
 ### sendPutRequest
 
@@ -142,9 +152,11 @@ I.sendPutRequest('/api/users.json', secret({ "email": "user@user.com" }));
 
 #### Parameters
 
--   `url` **[string][3]** 
+-   `url` **[string][4]** 
 -   `payload` **any** the payload to be sent. By default it is sent as an empty object 
--   `headers` **[object][2]** the headers object to be sent. By default it is sent as an empty object 
+-   `headers` **[object][3]** the headers object to be sent. By default it is sent as an empty object 
+
+Returns **[Promise][2]&lt;any>** response
 
 ### setRequestTimeout
 
@@ -160,6 +172,8 @@ I.setRequestTimeout(10000); // In milliseconds
 
 [1]: https://github.com/axios/axios
 
-[2]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object
+[2]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise
 
-[3]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String
+[3]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object
+
+[4]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String

--- a/lib/helper/REST.js
+++ b/lib/helper/REST.js
@@ -75,6 +75,8 @@ class REST extends Helper {
    * Executes axios request
    *
    * @param {*} request
+   *
+   * @returns {Promise<*>} response
    */
   async _executeRequest(request) {
     const _debugRequest = { ...request };
@@ -143,6 +145,8 @@ class REST extends Helper {
    *
    * @param {*} url
    * @param {object} [headers={}] - the headers object to be sent. By default it is sent as an empty object
+   *
+   * @returns {Promise<*>} response
    */
   async sendGetRequest(url, headers = {}) {
     const request = {
@@ -166,6 +170,8 @@ class REST extends Helper {
    * @param {*} url
    * @param {*} [payload={}] - the payload to be sent. By default it is sent as an empty object
    * @param {object} [headers={}] - the headers object to be sent. By default it is sent as an empty object
+   *
+   * @returns {Promise<*>} response
    */
   async sendPostRequest(url, payload = {}, headers = {}) {
     const request = {
@@ -197,6 +203,8 @@ class REST extends Helper {
    * @param {string} url
    * @param {*} [payload={}] - the payload to be sent. By default it is sent as an empty object
    * @param {object} [headers={}] - the headers object to be sent. By default it is sent as an empty object
+   *
+   * @returns {Promise<*>} response
    */
   async sendPatchRequest(url, payload = {}, headers = {}) {
     const request = {
@@ -228,6 +236,8 @@ class REST extends Helper {
    * @param {string} url
    * @param {*} [payload={}] - the payload to be sent. By default it is sent as an empty object
    * @param {object} [headers={}] - the headers object to be sent. By default it is sent as an empty object
+   *
+   * @returns {Promise<*>} response
    */
   async sendPutRequest(url, payload = {}, headers = {}) {
     const request = {
@@ -254,6 +264,8 @@ class REST extends Helper {
    *
    * @param {*} url
    * @param {object} [headers={}] - the headers object to be sent. By default it is sent as an empty object
+   *
+   * @returns {Promise<*>} response
    */
   async sendDeleteRequest(url, headers = {}) {
     const request = {

